src/components/ResourceCard/EditResource/ResourceDescribeActivity.js

```diff
@@ -27,19 +27,15 @@ export const uploadThumb = async (e, props) => {
     formData.append('thumb', e.target.files[0]);
     await props.uploadResourceThumbnail(formData);
   } catch (err) {
-    Swal.fire({
-      icon: 'error',
-      title: 'Error',
-      text: 'Failed to upload thumb.',
-    });
+    console.log(err);
```

Comment:
Same here

Reply:
it is being taken care from  actions/services